### DISCUSSION TITLE
Logging in using client ID and Client Secret to Azure OpenAI

### DISCUSSION THREAD

**Discussion by pkakathk** [BerriAI/litellm#1732] [Q&A] [answered]
Logging in using client ID and Client Secret to Azure OpenAI while using the proxy server. Currently the option available is to use Azure Vault. In our case we do not have Azure Vault. We use Client ID and Client Secret to get the Bearer token. Currently this function is not available with proxy server.
Any thoughts ? @ishaan-jaff and @krrishdholakia

**Comment by ishaan-jaff** [comment]
Hi @ppk5 thanks for this - can we hop on a call to better understand your problem ? 

Sharing my calendly for your convenience: https://calendly.com/d/4mp-gd3-k5k/litellm-1-1-onboarding-chat

**Comment by Manouchehri** [comment, marked as answer]
This was added in https://github.com/BerriAI/litellm/pull/3507.
